flake.nix

```diff
@@ -4,7 +4,12 @@
   # Setup trusted binary caches
   nixConfig = {
     trusted-substituters =
-      [ "https://cache.nixos.org/" "https://deku.cachix.org" ];
+      [ "https://cache.nixos.org/" "https://deku.cachix.org" "https://anmonteiro.cachix.org" ];
+    trusted-public-keys = [
+      "cache.nixos.org-1:6NCHdD59X431o0gWypbMrAURkbJ16ZPMQFGspcDShjY="
+      "deku.cachix.org-1:9zYY9BUBDV9xiXj8C0uCy+hkgjyDVRdH932KHmqVwYg="
+      "anmonteiro.cachix.org-1:KF3QRoMrdmPVIol+I2FGDcv7M7yUajp4F2lt0567VA4="
```

Comment:
As much as I greatly appreciate Antonio's work and use it for personal projects, I don't think we should allow it for deku.

Reply:
Sure, was just thinking that we might get fewer cache misses with it. But in theory everything we need is in our own cache.

If someone wants to lend this now feel free to fix it on this branch, otherwise I'll do it tomorrow